tracing-appender/README.md

```diff
@@ -0,0 +1,130 @@
+# tracing-appender
+
+Writers for logging events and spans
+
+[![Crates.io][crates-badge]][crates-url]
+[![Documentation][docs-badge]][docs-url]
+[![Documentation (master)][docs-master-badge]][docs-master-url]
+[![MIT licensed][mit-badge]][mit-url]
+[![Build Status][actions-badge]][actions-url]
+[![Discord chat][discord-badge]][discord-url]
+
+[Documentation][docs-url] | [Chat][discord-url]
+
+[crates-badge]: https://img.shields.io/crates/v/tracing-appender.svg
+[crates-url]: https://crates.io/crates/tracing-appender/0.1.0
+[docs-badge]: https://docs.rs/tracing-appender/badge.svg
+[docs-url]: https://docs.rs/tracing-appender/0.1.0
+[docs-master-badge]: https://img.shields.io/badge/docs-master-blue
+[docs-master-url]: https://tracing-rs.netlify.com/tracing-appender
+[mit-badge]: https://img.shields.io/badge/license-MIT-blue.svg
+[mit-url]: LICENSE
```

Comment:
This link won't work, since there isn't a `LICENSE` file in this directory. I believe this needs to be 
```suggestion
[mit-url]: ../LICENSE
```
(but don't quote me on this)

Reply:
Oh I see, I had copied an pasted from one of the other crates which used ../LICENSE. I see that it also exists at the root dir.